set.mm

```diff
@@ -353435,9 +353453,18 @@ the complex square root function (the branch cut is in the same place for
 
   ${
     $d a b $.
-    $( There exist irrational numbers ` a ` and ` b ` such that ` ( a ^ b ) `
+    $( There exist irrational numbers ` a ` and ` b ` such that ` ( a ^c b ) `
        is rational.  Statement in the Metamath book, section 1.1.5, footnote 27
-       on page 17.  (Contributed by AV, 23-Dec-2022.) $)
+       on page 17, and the "classical proof" for theorem 1.2 of [Bauer],
+       p. 483.  This proof is not acceptable in intuitionistic logic, since it
+       is based on the law of excluded middle:  Either
+       ` ( ( sqrt `` 2 ) ^c ( sqrt `` 2 ) ) ` , is rational, then
+       ` ( sqrt `` 2 ) ` are appropriate irrational numbers (see ~ sqrt2irr ),
+       otherwise ` ( ( sqrt `` 2 ) ^c ( sqrt `` 2 ) ) ` and ` ( sqrt `` 2 ) `
+       are appropriate irrational numbers, since
+       ` ( ( ( sqrt `` 2 ) ^c ( sqrt `` 2 ) ) ^c ( sqrt `` 2 ) ) = 2 ` .  For
```

Comment:
I would write:
> either s2 ^ s2 is rational, in which case s2, being irrational, can be chosen for both a and b, or s2 ^ s2 is irrational, in which case s2 ^ s2 can be chosen for a and s2 for b, since (s2 ^ s2) ^ s2 = 2 is rational.

Reply:
Done